tiledb/sm/storage_manager/context.h

```diff
@@ -111,27 +105,63 @@ class Context {
   /** The class logger. */
   shared_ptr<Logger> logger_;
 
+  /** The storage manager. */
+  StorageManager* storage_manager_;
```

Comment:
Is there a reason to allocate this member variable any more? I've not done a full audit, but it looks like allocation is only there to support the old non-C.41 `init()` function.

Reply:
I have a new story about this.  I propose making storage_manager_ C.41 and an actual member of context so that it will get cleaned up with RAII.  When I started making that change there were a few complications (other non-C.41 classes had to be dealt with, etc.)  I would prefer to defer this to that story.

Reply:
OK. Have to cut off scope creep somewhere.